src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+
+    operate(callback, number) {
+      if (typeof callback === `function`) {
+        result = callback(result, number);
+      }
+
+      return this;
+    },
+
+    reset() {
+      result = 0;
+
+      return this;
+    },
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      if (b !== 0) {
+        return a / b;
+      }
+
+      return 'false';
```

Comment:
Returning the string 'false' when dividing by zero is not ideal. Consider returning a more appropriate value like `NaN` (Not-a-Number) or throwing an error to indicate that division by zero is undefined.